qubes/ext/core_features.py

```diff
@@ -36,7 +36,8 @@ async def qubes_features_request(self, vm, event, untrusted_features):
 
         requested_features = {}
         for feature in (
-                'qrexec', 'gui', 'gui-emulated', 'qubes-firewall', 'vmexec'):
+                'qrexec', 'gui', 'gui-emulated', 'qubes-firewall', 'vmexec',
+                'pipewire'):
```

Comment:
You don't need to add it here. `supported-service.*` features are handled by "services" extension already. Here you'd handle plain `pipewire` but it isn't what you need.